app/move/views/_includes/assessment.njk

```diff
@@ -110,3 +114,5 @@
     {% endif %}
   {% endif %}
 {% endif %}
+{{ updateLink('court') }}
+</section>
```

Comment:
This file seems to be missing a new line, are you using `.editorconfig`? That usually sorts this issue automatically for me when saving files.

Reply:
Done